### PR TITLE
Post-release v1.0.6 changes

### PR DESCRIPTION
**Pull request by mmisol**
Updated version number and description in `pkg.json`.
Small change to `csproj` file to publish on `Release` build.

### CODE REVIEW

File: TuneUp/manifests/pkg.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+    "dependencies": [],
+    "host_dependencies": [],
+    "contents": "",
+    "engine_version": "2.5.0",
+    "engine": "dynamo",
+    "engine_metadata": "",
+    "site_url": "https://dynamobim.org/",
+    "repository_url": "https://github.com/DynamoDS/TuneUp",
+    "contains_binaries": true,
+    "node_libraries": []
```
Comment (QilongTang) [inline TuneUp/manifests/pkg.json:21]: @mmisol Can you check your VSCode setting for what's used for tabbing? Spaces or tabs?

File: TuneUp/TuneUp.csproj [hunk 2]
```diff
@@ -118,7 +118,7 @@
     <Copy SourceFiles="@(PackageJson)" DestinationFolder="$(PackageFolder)" />
     <MakeDir Directories="$(ExtraFolder)" Condition="!Exists($(ExtraFolder))"/>
     <MakeDir Directories="$(DyfFolder)" Condition="!Exists($(DyfFolder))"/>
-    <CallTarget Condition="'$(Configuration)' == 'Debug'" Targets="PackageDeploy" />
+    <CallTarget Condition="'$(Configuration)' == 'Release'" Targets="PackageDeploy" />
```
Comment (QilongTang) [inline TuneUp/TuneUp.csproj:121]: Forgot to clarify.. I don't think we want to commit this one, the change here is for publisher only

File: TuneUp/manifests/pkg.json (hunk 1 above)
Comment (mmisol) [reply TuneUp/manifests/pkg.json]: Sorry, they were spaces but my VS decided to format with 4 of them instead of 2. Fixed it

**Comment by QilongTang** [comment]
LGTM

**Comment by Amoursol** [comment]
Hello @mmisol,

The package **description** is missing two key pieces of information - can you please re-upload?

image

1. We need a statement at the top of the description(Description was accurate for the previous upload by @QilongTang):

*** TUNEUP WILL ONLY WORK IN DYNAMO 2.5 AND ABOVE AS IT UTILIZES NEW API FEATURES ***

2. The TuneUp blog link at the bottom of the description: https://dynamobim.org/tuneup-extension-explore-your-node-and-graph-execution-times/

**Comment by mmisol** [comment]
@Amoursol Thanks for finding this. It seems to be something we forgot to add back to this file from a previous release, so it got lost with the new release. I'll make sure to add it here this time.

**Comment by QilongTang** [comment]
@mmisol You can do so by downloading the previous version of TuneUp 1.0.5 and navigate to the package and find the the package.json to copy stuff from it or any other ways you prefer!

**Pull request merged**